Docker/README.md

```diff
@@ -655,7 +692,7 @@ CONTAINER ID        IMAGE               COMMAND                CREATED
 04d9c18da22a        c1c58f087482        "/bin/sh -c '/usr/sb   4 seconds ago       Up 3 seconds        0.0.0.0:80->80/tcp   thirsty_yalow
 ```
 
-Now that we have exposed the port, we're able to launch our container in daemon mode (-d) and by redirecting the local port 80 to the container port 80 on which our web server is listening. Try now reaching again your webserver on the local host (use your browser and point it to http://10.3.222.X if we host the lab or http://localhost if you run it locally). You should see a CentOS based page on your host distribution.
+Now that we have exposed the port, we're able to launch our container in daemon mode (-d) and redirect the local port 80 to the container port 80. Our web server is listening on port 80 of the container. Repeat the earlier attempt to connect to the web server in the container. You should see a CentOS based page on your host distribution.
```

Comment:
Humm I'd still say then "Our web server is listening on port 80 *in* the container (the container doesn't listen to anything itself).